tests/test_meshtastic_utils_messages.py

```
@@ -50,6 +66,153 @@
 TEST_PACKET_RX_TIME = 1234567890
 
 
+import mmrelay.meshtastic_utils as mu
+
+
+def _base_config():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_base_config`

(ANN202)